.travis.yml

```diff
@@ -37,6 +37,12 @@ matrix:
     - php: 7.0
       env: PHPCS=1 DEFAULT=0
 
+    - php: 7.0
```

Comment:
Yes testing against PHP 7 and 7.1 doesn't seem necessary especially when current CakePHP version support PHP 5.5+ and we don't use any PHP 7 features.